docs/release-notes/release-notes-0.19.0.md

```diff
@@ -433,6 +433,9 @@ The underlying functionality between those two options remain the same.
   inputs spending logic in the sweeper so it can properly handle missing inputs
   and recover from restart.
 
+* A code refactor that [refactor: use maps.Copy for cleaner map 
```

Comment:
nit: How about "A code refactor to use `maps.Copy` instead of manually copying map elements"